enos/enos-scenario-dr-replication.hcl

```diff
@@ -1300,6 +1302,34 @@ scenario "dr_replication" {
       vault_root_token        = step.create_secondary_cluster.root_token
       verify_pki_certs        = false
       verify_aws_engine_creds = false
+      verify_ssh_secrets      = false
+    }
+  }
+
+  step "verify_secrets_engines_delete" {
+    description = global.description.verify_secrets_engines_delete
+    module      = module.vault_verify_secrets_engines_delete
+    depends_on = [
+      step.verify_secrets_engines_on_primary,
+      step.verify_failover_replicated_data
+    ]
+
+    providers = {
+      enos = local.enos_provider[matrix.distro]
+    }
+
+    verifies = [
+      quality.vault_api_ssh_role_delete
+    ]
+
+    variables {
+      create_state       = step.verify_secrets_engines_on_primary.state
+      hosts              = step.get_secondary_cluster_ips.follower_hosts
+      leader_host        = step.get_secondary_cluster_ips.leader_host
+      vault_addr         = step.create_secondary_cluster.api_addr_localhost
+      vault_install_dir  = global.vault_install_dir[matrix.artifact_type]
+      vault_root_token   = step.create_secondary_cluster.root_token
+      verify_ssh_secrets = false
```

Comment:
I'm of many minds about this. On one hand, adding the step but turning off all of the actual verification future proofs calling the module if/when we add delete verification to other secrets engines.

On the other, we're effectively adding a step here that does nothing but appears to be doing something.

At the very least we need a comment explaining why we're not verifying the delete on DR clusters. But I also wonder if it would be better to run this against the _primary_ cluster and verify that delete works there, since we still want to verify the functionality, even if DR clusters don't handle client requests.